bofire/data_models/outlier_detection/outlier_detection.py

```diff
@@ -0,0 +1,21 @@
+from typing import Literal
+
+from pydantic import Field
+from typing_extensions import Annotated
+
+from bofire.data_models.base import BaseModel
+from bofire.data_models.surrogates.api import SingleTaskGPSurrogate
+
+
+class OutlierDetection(BaseModel):
+    type: str
+
+
+class IterativeTrimming(OutlierDetection):
```

Comment:
Please add a docstring in which you mention the method and the paper and explain the parameters.

Reply:
Please use google type docstrings